Sources/Commands/Options.swift

```diff
@@ -46,33 +46,43 @@ struct GlobalOptions: ParsableArguments {
 struct LocationOptions: ParsableArguments {
     init() {}
 
-    /// The custom build directory, if provided.
-    @Option(help: "Specify build/cache directory")
-    var buildPath: AbsolutePath?
+    /// The custom .build directory, if provided.
+    @Option(name: .customLong("scratch-space-path"), help: "Specify the scratch space directory (default .build)", completion: .directory)
```

Comment:
This name seems unfortunate to me but I don't have a better suggestion right now.  I'll try to come up with one and suggest it here.  I agree that "build" is too narrow here for what it does.

Reply:
right, I am not 100% set on it. `build` is too narrow and confusion. `workspace` could work also

Reply:
Yes, I think that would be better than "scratch-space".